fastf1/core.py

```diff
@@ -2844,18 +2844,32 @@ def pick_fastest(self, only_by_time: bool = False) -> "Lap":
         Returns:
             instance of :class:`Lap`
         """
+        # TODO: Deprecate returning empty lap object when there is no lap
+        # that matches definion
         if only_by_time:
             laps = self  # all laps
         else:
             # select only laps marked as personal fastest
-            laps = self.loc[self['IsPersonalBest'] == True]  # noqa: E712 comparison with True
+            laps = self.loc[self['IsPersonalBest'] == True]  # noqa: E712
```

Comment:
can you add the comment back? rarely anybody knows the meaning of E712

Reply:
Would the character limit. Let me know if you want to bypass it for this

Reply:
True, sadly, flake8 doesn't support stuff like disable for next line. Just leave it as it is then. I guess if someone wants to know, they need to look it up.